site/app/handlers.js

```diff
@@ -19,6 +23,9 @@ const extractArticleName = function (path) {
   return false
 }
 
+let isTokenValid = (date) => 
```

Comment:
Note that if someone managed to generate a token for the future it would work for a long time (until our secret refreshed).  Having a lower bound is useful to protect against this - i.e. min < delta < max.  Having min slightly below zero is good to handle clock drifts across different instances of the service (not much of an issue for us in a single deployment domain, but I tend to put it for this sort of stuff since it's easy at the outset)